crates/sui-core/src/checkpoints/checkpoint_executor/mod.rs

```diff
@@ -352,12 +355,7 @@ impl CheckpointExecutor {
         );
 
         let tx_count = execution_digests.len();
-        debug!(
-            epoch=?epoch_store.epoch(),
-            checkpoint_sequence=?checkpoint.sequence_number(),
-            "Number of transactions in the checkpoint: {:?}",
-            tx_count
-        );
+        debug!("Number of transactions in the checkpoint: {:?}", tx_count);
```

Comment:
Same here

Reply:
This function `execute_checkpoint()` is called by `schedule_checkpoint()`, which annotates the log with sequence num and epoch.